lib/autoyast.pm

```diff
@@ -221,7 +221,8 @@ sub expand_template {
         get_var => \&get_var,
         # pass reference to check_var
         check_var => \&check_var,
-        is_ltss => get_var('SCC_REGCODE_LTSS') ? '1' : '0'
+        is_ltss => get_var('SCC_REGCODE_LTSS') ? '1' : '0',
```

Comment:
Are mutually exclusive? What is going to happens if both `SCC_REGCODE_LTSS` and `SCC_REGCODE_LTSS_ES` are enabled at the same time?

Reply:
it is very hard to handle **is_ltss** and **is_ltss_es** once `SCC_REGCODE_LTSS` and `SCC_REGCODE_LTSS_ES` are enabled at the same time, we don't know people want what to do in this case.

Reply:
I plan to revert this modification, and update `autoyast_sles4sap_saptune.xml.ep` file like this:
```
      % if ($get_var->('SCC_REGCODE_LTSS')) {
      <addon>
        <!-- SUSE Linux Enterprise Server LTSS -->
        <name>SLES-LTSS</name>
        <version>{{VERSION}}</version>
        <arch>{{ARCH}}</arch>
        <reg_code><%= $get_var->('SCC_REGCODE_LTSS') %></reg_code>
      </addon>
      %}
      % if ($get_var->('SCC_REGCODE_LTSS_ES')) {
      <addon>
        <!-- SUSE Linux Enterprise Server LTSS Extended Security -->
        <name>SLES-LTSS-Extended-Security</name>
        <version>{{VERSION}}</version>
        <arch>{{ARCH}}</arch>
        <reg_code><%= $get_var->('SCC_REGCODE_LTSS_ES') %></reg_code>
      </addon>
      %}
```
this is not related to **is_ltss** or **is_ltss_es**